### PR TITLE
Add highlighting for selected thread state slices

### PR DESCRIPTION
**Pull request by Mahmood-Darwish**
Selected thread state slices are now highlighted with a white outline around them.

Bug: http://b/236945046

Screenshot: http://screenshot/7CMYJLydwTzopRE.png

### CODE REVIEW

File: src/OrbitGl/PrimitiveAssembler.cpp [hunk 1]
```diff
@@ -262,7 +262,7 @@ void PrimitiveAssembler::AddVerticalArrow(Vec2 starting_pos, Vec2 arrow_body_siz
       Vec2(starting_pos[0] - head_half_width, body_head_meeting_y),
       Vec2(starting_pos[0] + head_half_width, body_head_meeting_y),
   };
-  AddTriangle(arrow_head, z, color);
+  AddTriangle(arrow_head, z, arrow_color);
```
Comment (beckerhe) [inline src/OrbitGl/PrimitiveAssembler.cpp:265]: Same here.

File: src/OrbitGl/PrimitiveAssembler.cpp [hunk 2]
```diff
@@ -275,7 +275,7 @@ void PrimitiveAssembler::AddVerticalArrow(Vec2 starting_pos, Vec2 arrow_body_siz
       Vec2(starting_pos[0] + body_half_width, arrow_body_max_y),
   };
   Quad arrow_body(box_vertices);
-  AddBox(arrow_body, z, color);
+  AddBox(arrow_body, z, arrow_color);
```
Comment (beckerhe) [inline src/OrbitGl/PrimitiveAssembler.cpp:278]: Same here.

File: src/OrbitGl/PrimitiveAssembler.h [hunk 3]
```diff
@@ -114,7 +114,7 @@ class PrimitiveAssembler {
   //
   // x is starting_pos
   void AddVerticalArrow(Vec2 starting_pos, Vec2 arrow_body_size, Vec2 arrow_head_size, float z,
-                        const Color& color, ArrowDirection arrow_direction);
+                        const Color& arrow_color, ArrowDirection arrow_direction);
```
Comment (beckerhe) [inline src/OrbitGl/PrimitiveAssembler.h:117]: This is also not related to this change.

File: src/OrbitGl/ThreadStateBar.cpp [hunk 4]
```diff
@@ -237,6 +239,22 @@ void ThreadStateBar::DoUpdatePrimitives(PrimitiveAssembler& primitive_assembler,
         const Vec2 pos{x0, GetPos()[1]};
         const Vec2 size{width, GetHeight()};
 
+        if (slice == app_->selected_thread_state_slice()) {
+          const Color kWhiteBorder = Color(255, 255, 255, 255);
```
Comment (beckerhe) [inline src/OrbitGl/ThreadStateBar.cpp:243]: Nit: kBorderColor (Having the color itself in the name defies the purpose of the constant a bit)

**Comment by Mahmood-Darwish** [comment]
Selected thread state slices are now highlighted with a white outline around them.

Bug: http://b/236945046

Screenshot: http://screenshot/7CMYJLydwTzopRE.png
